src/components/scenes/CoinRankingScene.tsx

```diff
@@ -168,8 +223,8 @@ const CoinRankingComponent = (props: Props) => {
             start += QUERY_PAGE_SIZE
           }
           setDataSize(coinRankingDatas.length)
-          if (lastUsedFiat !== defaultIsoFiat) {
-            setLastUsedFiat(defaultIsoFiat)
+          if (lastFetchedFiat !== supportedFiatSetting && COINGECKO_SUPPORTED_FIATS.includes(supportedFiatSetting)) {
```

Comment:
the `COINGECKO_SUPPORTED_FIATS.includes(supportedFiatSetting)` check shouldn't be needed as it either is supported or it's USD